### DISCUSSION TITLE
elasticsearch hitting RLIMIT_MEMLOCK

### DISCUSSION THREAD

**Discussion by Stefomat** [nextcloud/all-in-one#3926] [Questions] [answered]
Every time elasticsearch starts, it hits the RLIMIT_MEMLOCK limit, which is set to 65536 by default. This can be seen as a log entry in dmesg.

`denied resource overstep by requesting 4230225920 for RLIMIT_MEMLOCK against limit 65536 for /usr/share/elasticsearch/jdk/bin/java[java:3748] uid/euid:1000/1000 gid/egid:0/0, parent /usr/share/elasticsearch/jdk/bin/java[java:2524] uid/euid:1000/1000 gid/egid:0/0`

How can I increase the limit accordingly? Does this have to be set inside or outside the container?
I found [this](https://www.elastic.co/guide/en/elasticsearch/reference/8.11/_memory_lock_check.html), which advises you to set the limit to unlimited.

Thanks

**Comment by szaimen** [comment, marked as answer]
I've found this: https://github.com/freedomofpress/ansible-role-elk/issues/42

**Comment by Stefomat** [comment]
To be honest, I don't know how I can use this for AIO. Config files are mentioned which, as I understand it, should be inside the container. But there must also be something outside the container, because that's where I get the error messages.
For now, I will just continue to ignore the "error messages" until a solution is found by someone with more knowledge than me.
